### ISSUE TITLE
Escape sequence makes selene panic

### ISSUE THREAD

**Issue by jeparlefrancais** [Kampfkarren/selene#205]
Using an escape sequence like this will make selene panic (but not exit with an error code, as described in #203):

```lua
return '\97b'
```

The reference manual explains the syntax of escape sequences in section `2.1 - Lexical Conventions`: https://www.lua.org/manual/5.1/manual.html

In summary: when using `\` followed by a digit, it takes up to the next three digits and that makes the byte value in decimal (not octal, as some stack overflow misleading answers seem to think)

**Comment by matthargett** [comment]
I think this is fixed in latest full-moon and therefore Selene release.

**Comment by Kampfkarren** [comment]
Nah, it's a part of a lint that checks for valid escape sequences. I forget why, but I'm pretty sure it's trying to parse something as a number, then panicking.
